spec/features/customer_choose_plan_spec.rb

```diff
@@ -0,0 +1,56 @@
+require 'rails_helper'
+
+feature 'Customer choose a plan' do
+  scenario 'successfully' do
+    yoga = ClassCategory.create!(name: 'Yoga')
+    first_plan = Plan.create!(name: 'PlanoFit', montlhy_rate: 200, monthly_class_limit: 5)
+    second_plan = Plan.create!(name: 'PlanoSmart', montlhy_rate: 300, monthly_class_limit: 7)
+    class_category_plan = ClassCategoryPlan.create!(plan: first_plan, class_category: yoga)
+    ccred = PaymentMethod.new(name: 'Cartão de Crédito', code: 'CCRED')
+    bol = PaymentMethod.new(name: 'Boleto', code: 'BOL')
+    payment_methods = [ccred, bol]
+    allow(PaymentMethod).to receive(:all).and_return(payment_methods)
```

Comment:
Seria interessante usar factory_bot nos testes, fiz uma sugestão nesse mas tem outros abaixo que seria legal trocar, o que acham?

```suggestion
    yoga = create(:class_category, name: 'Yoga')
    first_plan = create(:plan, name: 'PlanoFit', montlhy_rate: 200, monthly_class_limit: 5)
    second_plan = create(:plan, name: 'PlanoSmart', montlhy_rate: 300, monthly_class_limit: 7)
    class_category_plan = create(:class_category_plan, plan: first_plan, class_category: yoga)
    ccred = PaymentMethod.new(name: 'Cartão de Crédito', code: 'CCRED')
    bol = PaymentMethod.new(name: 'Boleto', code: 'BOL')
    allow(PaymentMethod).to receive(:all).and_return([ccred, bol])
```